src/config.cpp

```diff
@@ -709,19 +714,16 @@ config::attribute_value& config::operator[](config_key_type key)
 
 const config::attribute_value& config::get_old_attribute(config_key_type key, const std::string& old_key, const std::string& in_tag, const std::string& message) const
 {
-	if(has_attribute(old_key)) {
-		const std::string what = formatter() << "[" << in_tag << "]" << old_key << "=";
-		const std::string msg  = formatter() << "Use " << key << "= instead. " << message;
-		deprecated_message(what, DEP_LEVEL::INDEFINITE, "", msg);
-	}
-
 	attribute_map::const_iterator i = values_.find(key);
 	if(i != values_.end()) {
 		return i->second;
 	}
 
 	i = values_.find(old_key);
 	if(i != values_.end()) {
+		const std::string what = formatter() << "[" << in_tag << "]" << old_key << "=";
+		const std::string msg  = formatter() << "Use " << key << "= instead. " << message;
+		deprecated_message(what, DEP_LEVEL::INDEFINITE, "", msg);
```

Comment:
Why has this moved downwards? If the WML has a key with the old name then the warning should be shown; if it's using both the new name and the old name in the same WML tag, that's still a reason to show the warning.

Reply:
Looks like my comment on the same topic failed to be linked to the PR.

https://github.com/wesnoth/wesnoth/commit/cc7d549e507f48c42ea4c8c346c3950a4ab5e3c4#r146721338

Reply:
Reverted